src/components/input/input-default-styles.jsx

```diff
@@ -54,54 +54,54 @@ export default theme => {
   return {
     input: {
       ...mixins.basicBoxSizing,
-      color: palette.text.default,
+      color: palette.gray0,
       marginBottom: ({ label }) => bottomBorderSize - borderSize + (label ? inputMarginBottom : 0),
       position: 'relative',
-      '&.input--has-success': modifierFn(palette.variant.success),
-      '&.input--has-warning': modifierFn(palette.variant.warning),
+      '&.input--has-success': modifierFn(palette.positive),
+      '&.input--has-warning': modifierFn(palette.negative),
```

Comment:
`palette.index`

Reply:
no more index, just red for warnings and error